src/hooks/usePersistAppData.ts

```diff
@@ -3,7 +3,7 @@ import {STORAGE_KEY} from '../constants'
 import type {AppData} from '../utils/appData'
 
 export function usePersistAppData(data: AppData) {
-  const timeoutRef = useRef<number | null>(null)
+  const timeoutRef = useRef<NodeJS.Timeout | null>(null)
```

Comment:
I don't know... I thought about using this type when developing it and being in the NodeJS scope I chose not to use it